pkg/auth/service.go

```diff
@@ -1,5 +1,7 @@
 package auth
 
+//go:generate go run github.com/treeverse/lakefs/tools/wrapgen --package auth --output ./wrapper.gen.go --interface Service ./service.go
+//go:generate go run golang.org/x/tools/cmd/goimports@latest -w ./wrapper.gen.go
```

Comment:
Why is that for?
Which imports will this affect, in which file?

Reply:
*All* of them.  The Go parser is just that: it returns a very bare syntax tree with no analysis.  We parse service.go, which contains any number of imports.  But most of them are unrelated to the one interface we wrap.  Without this line, Go "helpfully" refuses to compile the code: it contains unused imports.

Go is so helpful, _there is no way to switch this off_, even for generated files.  The answer across all Go groups is always "run goimports".  So I did.

Reply:
Got it, that's actually a great solution, can you please add a comment for that? 
took me a while